### PR TITLE
Saeb dev 20220117 aa to azure python class

### PR DESCRIPTION
**Pull request by JuntaoZ-NC**
This PR is meant to be the first PR to provide step-by-step tutorial for the AA-to-Azure-Python-Wrapper Class created.

Here are the items included in this PR:

1. Readme files
2. Requirements file
3. sample codes
4. AA-to-Azure-Python-Wrapper Class itself

Readme files
1.General Readme that provides insights to the integration from AA to Azure finally Power BI
2.Azure specific Readme that shows step by step guidance related to Azure
2.PowerBI specific Readme that shows step by step guidance related to Power BI

Requirements file
requirements.txt file for python to install dependencies

sample codes
1.Jupyter notebook code sample to utilize the Class
2.ADF pipeline to integrate notebook with other activities

AA-to-Azure-Python-Wrapper Class
analytics_client Class that is designed to be imported as a wrapper to request AA data

### CODE REVIEW

**Review by Jordan-Morrison — APPROVED** [review]

**Review by alex-solo — APPROVED** [review]
LGTM

**Review by s-laugh — CHANGES_REQUESTED** [review]
The commits need to be signed (see here to update https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits), and this is removing a bunch of other files from other work.

**Comment by s-laugh** [comment]
Also you are adding a lot of sample code, which is great, but if it's already implemented in another repository (aka AP-Databricks) it would be better to link there instead of copying it here.

**Comment by JuntaoZ-NC** [comment]
updated ms.date and update readme to ref AP repo

File: Performance Dashboard.md [hunk 1]
```diff
@@ -1,100 +0,0 @@
-# Performance Dashboard
```
Comment (s-laugh) [inline Performance Dashboard.md:1]: This file shouldn't be deleted.

File: SAEB Architecture Design.md [hunk 2]
```diff
@@ -1,140 +0,0 @@
-# SAEB Architecture Design
```
Comment (s-laugh) [inline SAEB Architecture Design.md:1]: This file shouldn't be deleted.

File: SC Labs.md [hunk 3]
```diff
@@ -1,64 +0,0 @@
-# Lessons Learned
```
Comment (s-laugh) [inline SC Labs.md:1]: This file shouldn't be deleted.

**Review by s-laugh — CHANGES_REQUESTED** [review]
The commits are still not verified (PR can't be merged until they are).
There's still a number of files that I believe should be links to the repositories and not duplicated here - I commented on them individually.
This PR still is deleting some files that it shouldn't - I've also commented on those individually.

File: Adobe-Azure-analytics-api-v2.0/examples/ADFpipeline/oas_take_up.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
+{
```
[the remark this answers is not in the document]
Comment (JuntaoZ-NC) [reply Adobe-Azure-analytics-api-v2.0/examples/ADFpipeline/oas_take_up.json]: resolved here :https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/c6ab90ed62949e515b0febb7ad9e442bc8626196

File: Adobe-Azure-analytics-api-v2.0/examples/ADFpipeline/sclabs_site_analytics.json [hunk 5]
```diff
@@ -0,0 +1,93 @@
+{
```
[the remark this answers is not in the document]
Comment (JuntaoZ-NC) [reply Adobe-Azure-analytics-api-v2.0/examples/ADFpipeline/sclabs_site_analytics.json]: resolved here https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/c6ab90ed62949e515b0febb7ad9e442bc8626196

File: Adobe-Azure-analytics-api-v2.0/examples/DatabricksNotebook/Azurepipeline.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+
```
[the remark this answers is not in the document]
Comment (JuntaoZ-NC) [reply Adobe-Azure-analytics-api-v2.0/examples/DatabricksNotebook/Azurepipeline.md]: resolved here https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/c6ab90ed62949e515b0febb7ad9e442bc8626196

File: Adobe-Azure-analytics-api-v2.0/examples/JupyterNotebook/JWTExample.ipynb [hunk 7]
```diff
@@ -0,0 +1 @@
...ad_blob(data.to_csv(), overwrite=True)"]}],"metadata":{"application/vnd.databricks.v1+notebook":{"dashboards":[],"language":"python","notebookMetadata":{"pythonIndentUnit":4},"notebookName":"AA Ingestion Script","notebookOrigID":3213229525829659,"widgets":{"dimensions":{"currentValue":"","nuid":"858740b7-2454-4c6c-979a-25b9c9a98d58","widgetInfo":{"defaultValue":"","label":"","name":"dimensions","options":{"validationRegex":null,"widgetType":"text"},"widgetType":"text"}},"metrics":{"currentValue":"","nuid":"7395672d-fb33-4328-a4e0-4f05d9e1043f","widgetInfo":{"defaultValue":"","label":"","name":"metrics","options":{"validationRegex":null,"widgetType":"text"},"widgetType":"text"}},"output_file_path":{"currentValue":"","nuid":"bdb21acd-421c-4ecc-91f9-359edf37a1ab","widgetInfo":{"defaultValue":"","label":"","name":"output_file_path","options":{"validationRegex":null,"widgetType":"text"},"widgetType":"text"}},"segments":{"currentValue":"","nuid":"e440cb11-abb1-427d-a033-32f260b45862","widgetInfo":{"defaultValue":"","label":"","name":"segments","options":{"validationRegex":null,"widgetType":"text"},"widgetType":"text"}}}},"language_info":{"name":"python"}},"nbformat":4,"nbformat_minor":0}
```
[the remark this answers is not in the document]
Comment (JuntaoZ-NC) [reply Adobe-Azure-analytics-api-v2.0/examples/JupyterNotebook/JWTExample.ipynb]: No, we do not have Jupyter Notebooks in the cloud ,and I used Jupyter in my local machine ,thus removed sample code here :https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/c6ab90ed62949e515b0febb7ad9e442bc8626196

File: Performance Dashboard.md (hunk 1 above)
Comment (JuntaoZ-NC) [reply Performance Dashboard.md]: Apologies for almost losing 3 documentations due to rebasing on outdated branch ,
Restored 3 documentations here https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/75e8489d3e2d3650d968170672c5b9da3c254962
https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/4a190ef293362e9fd10a72afebb7ccbfd582e9c8

File: SAEB Architecture Design.md (hunk 2 above)
Comment (JuntaoZ-NC) [reply SAEB Architecture Design.md]: Apologies for almost losing 3 documentations due to rebasing on outdated branch ,
Restored 3 documentations here https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/75e8489d3e2d3650d968170672c5b9da3c254962
https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/4a190ef293362e9fd10a72afebb7ccbfd582e9c8

File: SC Labs.md (hunk 3 above)
Comment (JuntaoZ-NC) [reply SC Labs.md]: Apologies for almost losing 3 documentations due to rebasing on outdated branch ,
Restored 3 documentations here https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/75e8489d3e2d3650d968170672c5b9da3c254962
https://github.com/DTS-STN/CDP-Lessons-Learned/pull/11/commits/4a190ef293362e9fd10a72afebb7ccbfd582e9c8

**Comment by JuntaoZ-NC** [comment]
PR closed due to duplicate commit ,opened another PR https://github.com/DTS-STN/CDP-Lessons-Learned/pull/13

**Pull request closed without merging**